common/libs/values.ts

```diff
@@ -24,7 +24,9 @@ export function sanitizeHex(hex: string) {
   return hex !== '' ? `0x${padLeftEven(hexStr)}` : '';
 }
 
-export function networkIdToName(networkId: string | number): StaticNetworkIds {
+export function networkIdToName(
+  networkId: 1 | 3 | 4 | 42 | '1' | '3' | '4' | '42'
```

Comment:
Are we hardcoding the expected chainIds here? I don't like this for long-term maintainability reasons (multiple places to configure adding the chainId when adding a new network)

Reply:
The expected chainIds are already hard coded in the switch statement, so i just added type checking so that if you enter an invalid chainId you dont have to wait for the switch statement to throw an error before you find out

Reply:
Why are we hardcoding them in the switch statement? We should be able to traverse the network object and find the matching network.

Reply:
That's legacy code that's been there for awhile. Not sure why its there

Reply:
This type of approach would allow us to configure additional networks in only one place, which I think is valuable.

Reply:
I think we should refactor this now to make sure it won't bite us later.